ironfist/src/cpp/combat/creatures.h

```diff
@@ -110,6 +110,8 @@ enum CREATURE_FLAGS : __int16
 struct tag_monsterInfo
 {
   __int16 cost;
+  //__int16 secondary_cost_id;

```

Comment:
This was my first attempt. It definitely did not work. Let me remove these two commented lines.